src/main/java/org/openjdk/jextract/clang/libclang/Index_h.java

```diff
@@ -80,15 +82,28 @@ static MemoryLayout align(MemoryLayout layout, long align) {
         };
     }
 
+    static final SymbolLookup SYMBOL_LOOKUP;
 
     static {
-        String libName = System.getProperty("os.name").startsWith("Windows") ? "libclang" : "clang";
-        System.loadLibrary(libName);
+        String osName = System.getProperty("os.name");
+        if (osName.startsWith("AIX")) {
+            String javaHome = System.getProperty("java.home");
+            String libName = javaHome + "/lib/libclang.a";
+            String clangVersion = "14";
+            try (BufferedReader br = new BufferedReader(
+                new FileReader(javaHome + "/conf/jextract/libclang.version"))) {
+                clangVersion = br.readLine().trim();
+            } catch (Exception e) {
+                throw new IllegalStateException("Failed to read libclang.version", e);
+            }
```

Comment:
FWIW, `clangVersion` is definitely assigned after the try block, so a default value is not needed. Also, `Files::readString` makes this a little simpler:
```suggestion
            String clangVersion;
            try {
                clangVersion = Files.readString(Path.of(javaHome + "/conf/jextract/libclang.version"));
            } catch (Exception e) {
                throw new IllegalStateException("Failed to read libclang.version", e);
            }
```